config/imsim-config.yaml

```diff
@@ -230,6 +232,8 @@ output:
     nproc: 1    # Change this to work on multiple CCDs at once.
     nfiles: 1   # Default is all 189 CCDs.  Set to 1 while testing.
 
+    camera_class: lsst.obs.lsst.LsstCam
```

Comment:
I'd suggest probably have this be just
```
camera: LsstCam
```
here, and remove it entirely from input.  I think having it load at input time made sense when we were reading a pickle file, but probably doesn't anymore.  This seems like it should just be a value specification, not an input type now.